### ISSUE TITLE
[PrivateGPT] Container without model

### ISSUE THREAD

**Issue by SlowLogicBoy** [3x3cut0r/docker#6]
Have you considered making docker image that does not contain model itself?

1. Because container is very large.
2. It would also benefit those who want to use different models.
3. If no model specified and it does not exist download default on startup.

**Comment by 3x3cut0r** [comment]
yes i considered this already. but it is not that easy, because it is already loaded within the setup process in this step:
`poetry install --with local`

a workaround could be to set the default model to a much smaller one to reduce the image size.

or i could think about removing it after the setup. then the user would have to load it again himself afterwards. i would have to think about that again.

**Comment by 3x3cut0r** [comment]
I created a new version without a model. Its downloaded via the setup script.
Try it out.
